src/assets/styles/map.scss

```diff
@@ -1,8 +1,13 @@
 @import "~styles/variables";
 
-.map {
-  background: $blue;
-  width: 100%;
-  height: 100%;
-  position: absolute;
+#map {
```

Comment:
The absolute positioning stuff in here is borking everything in safari and FF, switched to flex. Also switched to ID selector cause mapbox kind of assumes that and then it inserts classes on the same element, and for us to override those classes requires a higher specificity (could also just do a two class selector but meh).